WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -77,94 +86,135 @@ function DelegatesOfRegion({ activeSeniorDelegate, delegates, isAdminMode }) {
             <Table.Cell>
               {I18n.t(`enums.user.delegate_status.${delegate.delegate_status}`)}
             </Table.Cell>
-            <Table.Cell>
-              {delegate.location}
-            </Table.Cell>
+            <Table.Cell>{delegate.location}</Table.Cell>
           </Table.Row>
         ))}
       </Table.Body>
     </Table>
   );
 }
 
-export default function Delegates({
-  delegates,
-  isEditVisible,
-}) {
-  const seniorDelegates = React.useMemo(() => delegates
-    .filter((user) => user.delegate_status === 'senior_delegate')
-    .sort((user1, user2) => (user1.location || '').localeCompare(user2.location || '')), [delegates]);
+export default function Delegates() {
+  const { data, loading, error } = useLoadedData(delegatesStaticPageDataUrl);
+
+  const { delegates, canViewDelegateMatters } = data || {};
+  const seniorDelegates = React.useMemo(
+    () => !!delegates
+      && delegates
+        .filter((user) => user.delegate_status === 'senior_delegate')
+        .sort((user1, user2) => (user1.location || '').localeCompare(user2.location || '')),
+    [delegates],
```

Comment:
Something like `if (!delegates) return [];` looks more readable. This is too dense.

Reply:
Done.